src/EnergyPlus/WaterCoils.cc

```diff
@@ -533,9 +533,9 @@ namespace WaterCoils {
 
 			WaterCoil( CoilNum ).DesTotWaterCoilLoad = NumArray( 3 );
 
-			if ( WaterCoil( CoilNum ).UACoil == AutoSize && WaterCoil( CoilNum ).CoilPerfInpMeth == UAandFlow ) WaterCoil( CoilNum ).RequestingAutoSize = true;
-			if ( WaterCoil( CoilNum ).MaxWaterVolFlowRate == AutoSize && WaterCoil( CoilNum ).CoilPerfInpMeth == UAandFlow ) WaterCoil( CoilNum ).RequestingAutoSize = true;
-			if ( WaterCoil( CoilNum ).DesTotWaterCoilLoad == AutoSize && WaterCoil( CoilNum ).CoilPerfInpMeth == NomCap ) WaterCoil( CoilNum ).RequestingAutoSize = true;
+			//FB if ( WaterCoil( CoilNum ).UACoil == AutoSize && WaterCoil( CoilNum ).CoilPerfInpMeth == UAandFlow ) WaterCoil( CoilNum ).RequestingAutoSize = true;
+			//FB if ( WaterCoil( CoilNum ).MaxWaterVolFlowRate == AutoSize && WaterCoil( CoilNum ).CoilPerfInpMeth == UAandFlow ) WaterCoil( CoilNum ).RequestingAutoSize = true;
+			//FB if ( WaterCoil( CoilNum ).DesTotWaterCoilLoad == AutoSize && WaterCoil( CoilNum ).CoilPerfInpMeth == NomCap ) WaterCoil( CoilNum ).RequestingAutoSize = true;
```

Comment:
@wfbuhl these were necessary before because an input file using Performance Input Method = UFactorTimesAreaAndDesignWaterFlowRate where U-Factor and Max Water Flow were not autosized did not require a Sizing:Plant object. Now some example files are showing a warning where a Sizing:Plant object could not be found. If these are not used then the Sizing routine needs to account for when a Sizing:Plant object is required and when it's not.